ircd/resources.yaml

```diff
@@ -65,13 +70,49 @@ spec:
   - name: irc
     port: 6697
     targetPort: 6697
-  - name: websockets
```

Comment:
I think you need to point port 443 at the ingress?

Reply:
also port 80 I guess?

Reply:
does that belong in the loadbalancer or the ingress?

Reply:
normally the ingress creates a load balancer *for* us; however for our case here we want to reuse an existing one